airflow/runtime_templates.go

```diff
@@ -0,0 +1,273 @@
+package airflow
```

Comment:
I feel like we are missing unit tests for all this new logic, could we add that please, it's more so important because the logic around extract and copy is not so straight forward.

Reply:
Updated the required tests now.